wallet/walletrpc.c

```diff
@@ -101,15 +99,16 @@ static void json_withdraw(struct command *cmd,
 	if (!param(cmd, buffer, params,
 		   p_req("destination", json_tok_tok, &desttok),
 		   p_req("satoshi", json_tok_tok, &sattok),
+		   p_opt("feerate", json_tok_feerate, &feerate_per_kw),
 		   NULL))
 		return;
 
 	if (!json_tok_wtx(&withdraw->wtx, buffer, sattok, -1ULL))
 		return;
 
 	if (!feerate_per_kw) {
-		command_fail(cmd, LIGHTNINGD, "Cannot estimate fees");
-		return;
+		if (!json_feerate_estimate(cmd, &feerate_per_kw, FEERATE_NORMAL))
```

Comment:
Was wondering why a seperate `json_feerate_estimate` here and not use a `withdraw_feerate`, similar to `opening_feerate`, `unilateral_feerate`, `mutual_close_feerate` etc.

Reply:
That would be good, too.  Will address.